sql/01_db_init.sql

```diff
@@ -45,3 +47,26 @@ create table bans_name
   ban_name         text not null,
   FOREIGN KEY (child_id) REFERENCES users (id)
 );
+
+create table home_points
+(
+  id              INTEGER UNIQUE PRIMARY KEY AUTOINCREMENT,
+  user_id         INTEGER,
+  name            TEXT NOT NULL,
+  points          INTEGER NOT NULL,
+  max_day         INTEGER NOT NULL,
+  max_week        INTEGER NOT NULL,
+  FOREIGN KEY (user_id) REFERENCES users (id)
+);
+
+create table prizes
+(
+  id              INTEGER UNIQUE PRIMARY KEY AUTOINCREMENT,
+  user_id         INTEGER,
+  name            TEXT NOT NULL,
+  points          INTEGER NOT NULL DEFAULT NULL,
```

Comment:
`DEFAULT NULL` tutaj nie zadziała, bo jest wcześniej `NOT NULL`

Reply:
Poprawiłam